app/src/main/res/layout/submitted_answer_item.xml

```diff
@@ -17,24 +23,33 @@
     android:layout_marginEnd="18dp"
     android:layout_marginStart="68dp">
 
-    <FrameLayout
+    <TextView
+      android:id="@+id/submitted_answer_text_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:background="@drawable/submitted_answer_background"
+      android:fontFamily="sans-serif"
+      android:paddingStart="16dp"
+      android:paddingTop="8dp"
+      android:paddingEnd="16dp"
+      android:paddingBottom="8dp"
+      android:text="@{submittedAnswer}"
+      android:textSize="16sp"
       app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintTop_toTopOf="parent">
+      app:layout_constraintTop_toTopOf="parent" />
 
-      <TextView
-        android:id="@+id/submitted_answer_text_view"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:fontFamily="sans-serif"
-        android:paddingStart="16dp"
-        android:paddingTop="8dp"
-        android:paddingEnd="16dp"
-        android:paddingBottom="8dp"
-        android:text="@{submittedAnswer}"
-        android:textSize="16sp" />
-    </FrameLayout>
+    <androidx.recyclerview.widget.RecyclerView
```

Comment:
We should default the visibility for this & the above (probably to gone).

Reply:
done

Reply:
Resolving on behalf of Ben